$OpenBSD: patch-pfe_Makefile_in,v 1.3 2008/10/22 12:25:50 jasper Exp $

Patch created without context (diff -C 0), as an RCS Id is found in
the context, which is expanded, while it shouldn't.

*** pfe/Makefile.in.orig	Wed Oct 22 14:23:25 2008
--- pfe/Makefile.in	Wed Oct 22 14:23:52 2008
***************
*** 327 ****
! DEBUG = @WITHOUT_OPT@ @DEBUG@
--- 327 ----
! DEBUG =
***************
*** 364 ****
! OPTIM = @WITH_OPT@    @OPTIM@
--- 364 ----
! OPTIM =
